Automattic/wp-calypso
# Hosting metrics: Requests per minute unification for line chart

- Related to https://github.com/Automattic/wp-calypso/pull/80712#pullrequestreview-1582638191
- Alternative of https://github.com/Automattic/wp-calypso/pull/80755

## Proposed Changes

* Use the same formula for all the line charts. The result from the API * 60 and do not use the `_meta.dimension`.
* The last HTTP status code charts will display tiny decimal numbers.
* Update the HTTP status code chart descriptions to say `requests per minute`.

<img width="1245" alt="CleanShot 2023-08-18 at 08 28 34@2x" src="https://github.com/Automattic/wp-calypso/assets/36432/9494a809-f544-4726-82c8-41fc5a5f6a6d">

**Context**

The API returns each data point (`value`) in requests per second, and also a general time resolution.
The time resolutions are:
| **Time picker** | **Resolution in seconds** |
|---|---|
| 6 hours | 240 |
| 24 hours | 900 |
| 3 days | 3600 |
| 7 days | 7200 |

I can see these different formulas:
- `value * resolution` : actual number of requests during the resolution (time window)
- `value * resolution / 60` : requests per minute during the resolution (time window)
- `value * 60`: requests in one single minute

This PR implements the last one. Number of requests in one single minute.

## Testing Instructions



* Navigate to `/site-monitoring/${siteSlug}`
* Observe the line charts display the label `Requests per minute` and the values are correct.


## Screenshots
<img width="955" alt="Screenshot 2023-08-18 at 16 24 27" src="https://github.com/Automattic/wp-calypso/assets/779993/f03c61d4-8052-4d61-85b6-9d8cb642fd9f">

<img width="951" alt="Screenshot 2023-08-18 at 16 24 40" src="https://github.com/Automattic/wp-calypso/assets/779993/3e3f1850-0db1-46e7-b054-93e73387e608">

<img width="949" alt="Screenshot 2023-08-18 at 16 25 17" src="https://github.com/Automattic/wp-calypso/assets/779993/6937301c-8e47-4038-93ac-16bab2ab90b2">




## Pre-merge Checklist
